internal/ports/grpc/ping.port.go

```diff
@@ -0,0 +1,18 @@
+package portsGRPC
+
+import (
+	"context"
```

Comment:
just for more pretty :v
```suggestion
	"context"
	"log"
	
	servicesHealthCheck "togo/internal/core/services/healthcheck"
	
	pb "togo/pkg/grpc"
	"google.golang.org/protobuf/types/known/timestamppb"
```
- the first part: package in go
- the 2nd: internal package
- the 3rd: external package: lib, framework etc